var/spack/repos/builtin/packages/spades/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
+import subprocess
+
+
+class Spades(Package):
+    """SPAdes - St. Petersburg genome assembler - is intended for both
+       standard isolates and single-cell MDA bacteria assemblies."""
+
+    homepage = "http://www.example.com"
+    url      = "http://cab.spbu.ru/files/release3.10.1/SPAdes-3.10.1.tar.gz"
+
+    version('3.10.1', 'dcab7d145af81b59cc867562f27536c3')
+
+    depends_on('python')
+    depends_on('cmake', type='build')
+    depends_on('zlib')
+    depends_on('bzip2')
+
+    conflicts('%gcc@7.1.0:')
+
+    def install(self, spec, prefix):
+        os.environ['PREFIX'] = prefix
+        subprocess.call(['./spades_compile.sh'])
```

Comment:
I took a look at this script and it just runs CMake like any other `CMakePackage` does, but without any of the flags that Spack adds by default. I think the best course of action would be to convert this package to `CMakePackage` and remove the `install()` function altogether.